### PR TITLE
COMMERCE-7203 Apply mvcc to products

### PR DESCRIPTION
**Pull request by ctang3**
https://issues.liferay.com/browse/COMMERCE-7203

Tests in `commerce-products-test` are all passing after https://github.com/Preston-Crary/liferay-portal/commit/77caaa706bff1fc30cc551ecd0671d7c278183c9

Running the others modules' tests on CI here

### CODE REVIEW

**Comment by ctang3** [comment]
ci:test

**Comment by ctang3** [comment]
@Preston-Crary applied changes, test is still passing 👍

**Comment by Preston-Crary** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/CommerceProductUpgradeStepRegistrator.java [hunk 1]
```diff
@@ -236,4 +259,4 @@ public void register(Registry registry) {
 	@Reference
 	private SettingsFactory _settingsFactory;
 
-}
\ No newline at end of file
+}
```
Comment (Preston-Crary) [inline modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/upgrade/CommerceProductUpgradeStepRegistrator.java:262]: Can you fixup this line - it shouldn't be modified.

**Comment by ctang3** [comment]
@Preston-Crary fixed SF,  thanks!

**Comment by ctang3** [comment]
@Preston-Crary woops actually fixed to use the new MVCCVersionUpgradeProcess this time

**Comment by Preston-Crary** [comment]
Please see https://github.com/Preston-Crary/liferay-portal/pull/1497

**Pull request closed without merging**